bigquery/deploy/README.md

```diff
@@ -0,0 +1,71 @@
+## Running on GKE
+
+* Run Elasticsearch on GKE
+  * Clone `kubernetes-elasticsearch-cluster` repo.
+    ```
+    git clone https://github.com/pires/kubernetes-elasticsearch-cluster.git
+    cd kubernetes-elasticsearch-cluster
+    ```
+  * Set up [Internal Load Balancer](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing).
+  Note: This is not needed for indexing. This will be needed for having API
+  server on App Engine Flex talk to Elasticsearch in GKE; might as well set up
+  now. Change `es-svc.yaml` to:
```

Comment:
We should consider just maintaining our own set of config yamls using kubernetes-elasticsearch-cluster as a starting point, and have our custom configs built in so that users don't have to mess with this directly.